### PR TITLE
Adding new feature functions

### PR DESCRIPTION
**Pull request by jbschiratti**
- Adding univariate and bivariate feature functions to mne_features + tests.
- Tests OK with Numba. Might not be OK yet without Numba.

### CODE REVIEW

File: mne_features/tests/test_bivariate.py [hunk 1]
```diff
@@ -13,8 +13,8 @@
                                     compute_time_corr_coefs)
 
 rng = np.random.RandomState(42)
-sfreq = 256.
-data = rng.standard_normal((10, 20, int(sfreq)))
+sfreq = 64.
+data = rng.standard_normal((3, 5, int(sfreq)))
```
Comment (agramfort) [inline mne_features/tests/test_bivariate.py]: @jbschiratti with such smaller data I get segfault with numba.

File: mne_features/mock_numba.py [hunk 2]
```diff
@@ -16,7 +16,7 @@ def __init__(self, **kwargs):  # noqa: D102
 
     class MockType(object):
         def __getitem__(self, slice):
-            return
+            return self
```
Comment (jbschiratti) [inline mne_features/mock_numba.py]: The fix is so simple, yet I could not figure it out... ! Now I get it.

File: mne_features/tests/test_bivariate.py (hunk 1 above)
Comment (jbschiratti) [reply mne_features/tests/test_bivariate.py]: Yes, it seems there is an issue with `compute_nonlinear_interdep`. I'll fix that.

**Comment by agramfort** [comment]
thanks @jbschiratti !

**Pull request merged**